src/translations/fr.json

```diff
@@ -157,20 +157,20 @@
   "nominalVoltage1": "Tension nominale 1",
   "nominalVoltage2": "Tension nominale 2",
   "nominalVoltage3": "Tension nominale 3",
-  "nominalVoltage1Or": "Tension nominale Or (kV)",
-  "nominalVoltage2Ex": "Tension nominale Ex (kV)",
-  "voltageLevelId1": "ID Poste Or",
-  "voltageLevelId2": "ID Poste Ex",
-  "terminal1Connected": "Connecté Or",
-  "terminal2Connected": "Connecté Ex",
-  "country1": "Pays Or",
-  "country2": "Pays Ex",
+  "nominalVoltage1KV": "Tension nominale 1 (kV)",
+  "nominalVoltage2KV": "Tension nominale 2 (kV)",
+  "voltageLevelId1": "ID Poste 1",
+  "voltageLevelId2": "ID Poste 2",
+  "terminal1Connected": "Connecté 1",
+  "terminal2Connected": "Connecté 2",
+  "country1": "Pays 1",
```

Comment:
There are already `Countries1` and `'Countries2` with the same value :)

Reply:
it doesn't have the same value in English

"Countries1": "Countries 1",
  "Countries2": "Countries 2",
  
"country1": "Country 1",
  "country2": "Country 2",

Reply:
my bad